index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flixter</title>
+
+    <link rel="stylesheet" href="style.css">
+    <script defer src="script.js"></script>
+</head>
+<body>
+
+    <h1>FLIXTER 🍿</h1>
+
+    <form id="search-form">
+        <label for="search"></label>
+            <input id="search-input" type="text" name="search" placeholder="🔍 Search a Movie...">
+            <button type="submit" id="search-button" >Search</button>
+            <button id="close-search-btn">Clear Results</button>
+    </form>
+
+    
+
+    <h2 id = "now-playing">Now Playing 🎞️ </h2>
+
```

Comment:
very minor note, but remove the spaces around the equal sign in `id = "now-playing"` (so it reads `id="now-playing"`)